shard.yml

```diff
@@ -38,8 +38,8 @@ dependencies:
     version: ~> 0.6.10
 
   teeplate:
-    github: mosop/teeplate
-    version: ~> 0.4.5
+    github: amberframework/teeplate
+    branch: master
```

Comment:
Why master?, I think we should create a tag for this 😅

Reply:
Yeah that makes sense.